prefixdb_test.go

```diff
@@ -49,7 +49,63 @@ func TestWithGolevelDB(t *testing.T) {
 
 	pdb := NewPrefixDB(db, bz("key"))
 
-	t.Run("PrefixDB", func(t *testing.T) { Run(t, pdb) })
+	t.Run("GolevelDB", func(t *testing.T) { Run(t, pdb) })
+}
+
+func TestWithClevelDB(t *testing.T) {
+	name := fmt.Sprintf("test_%x", randStr(12))
+	defer cleanupDBDir("", name)
+
+	db, err := NewCLevelDB(name, "")
+	require.Nil(t, err)
+
+	pdb := NewPrefixDB(db, bz("key"))
```

Comment:
I don't know that we need to test the prefixdb for _every_ backend. What I had in mind here was ensuring we had concurrency tests for all the backends, and then for the prefixed wrapper. (This isn't bad either, but it exercises two things at once)